### ISSUE TITLE
Bootstrap5 support?

### ISSUE THREAD

**Issue by gcatto** [sciactive/pnotify#415]
Are there any plans to add explicit Bootstrap5 support, similar to Bootstrap4?

**Comment by hperrin** [comment]
Yes. I'll have a chance to work on it soon, and I'll be adding a bootstrap5 module.

**Comment by gcatto** [comment]
Great! Is there something I can do to help? I could make a branch where I copy the Bootstrap4 package and update it for Bootstrap5?

**Comment by DevProLog** [comment]
Yes, please add it soon.

**Comment by phoenixcoded20** [comment]
Do we have any ETA on this?

**Comment by brunoferreiradev07** [comment]
@hperrin Are there any updates on this issue?

**Comment by gotofil** [comment]
> @hperrin Are there any updates on this issue?

Kind of: https://github.com/sciactive/pnotify/issues/420#issue-2041787412

**Comment by hperrin** [comment]
Sorry everyone, I'll make this a priority when I work on PNotify next.

**Comment by interceptor** [comment]
Is this nice look project still alive?
